### PR TITLE
Enhancing Shell Section ElasticMembranePlateSection

### PR DESCRIPTION
**Pull request by p-ranchal**
The current elastic shell section [_ElasticMembranePlateSection_](https://opensees.berkeley.edu/wiki/index.php/Elastic_Membrane_Plate_Section) uses the same Young's modulus for in-plane (membrane) and out-of-plane (plate) actions. As practitioners, we typically need to be able to model slabs with different in-plane and out-of-plane elastic stiffness parameters. This pull request modifies _ElasticMembranePlateSection_  so that it allows the use of different Young's moduli for membrane and plate actions.

This section has been successfully tested with [ShellMITC4](https://opensees.berkeley.edu/wiki/index.php/Shell_Element), [ShellDKGQ](https://opensees.berkeley.edu/wiki/index.php/ShellDKGQ) and [ASDShellQ4](https://github.com/OpenSees/OpenSees/pull/365) elements.

### CODE REVIEW

**Comment by mhscott** [comment]
How about overloading the constructor and adding another E value as private data for the current ElasticMembranePlaneSection?

File: SRC/interpreter/OpenSeesSectionCommands.cpp [hunk 1]
```diff
@@ -81,7 +81,6 @@ void* OPS_ElasticTubeSection3d();
 void* OPS_ParallelSection();
 void* OPS_SectionAggregator();
 void* OPS_ElasticPlateSection();
-void* OPS_ElasticMembranePlateSection();
```
Comment (p-ranchal) [inline SRC/interpreter/OpenSeesSectionCommands.cpp:84]: Duplicate of line 77

File: SRC/interpreter/OpenSeesSectionCommands.cpp [hunk 2]
```diff
@@ -1053,7 +1052,6 @@ namespace {
 	functionMap.insert(std::make_pair("Aggregator", &OPS_SectionAggregator));
 	functionMap.insert(std::make_pair("AddDeformation", &OPS_SectionAggregator));
 	functionMap.insert(std::make_pair("ElasticPlateSection", &OPS_ElasticPlateSection));
-	functionMap.insert(std::make_pair("ElasticMembranePlateSection", &OPS_ElasticMembranePlateSection));
```
Comment (p-ranchal) [inline SRC/interpreter/OpenSeesSectionCommands.cpp:1056]: Duplicate of line 1041

File: SRC/element/fourNodeQuad/SixNodeTri.cpp [hunk 3]
```diff
@@ -1236,12 +1239,12 @@ SixNodeTri::getResponse(int responseID, Information &eleInfo)
       cnt += 3;
     }
 
-	double We[nnodes][nip] = {{1.6666666666666667, -0.3333333333333333, -0.3333333333333333},
```
Comment (p-ranchal) [inline SRC/element/fourNodeQuad/SixNodeTri.cpp:1239]: This wouldn't compile for me so replaced the variable with literals.

**Comment by p-ranchal** [comment]
Dear Prof. Scott,

I have refactored ElasticMembranePlateSection.cpp/h files and eliminated the ElasticMembranePlateSection2.cpp/h files per your suggestion. I did not need to overload the existing constructor for ElasticMembranePlateSection and instead added an optional out-of-plane stiffness modification parameter similar to the currently optional mass density parameter rho. 

This is implementation is leaner, also maintains backward compatibility, and is also readily extensible in case we need to add additional stiffness modifiers in the future. I have retested this implementation with ShellMITC4, ShellDKGQ, and ASDShellQ4 elements.

Thanks,

Pearl

**Comment by p-ranchal** [comment]
Dear @mhscott , 

Just wanted to follow up to see if you have any further comments on this or can this be merged?

Thanks,

Pearl

**Comment by mhscott** [comment]
Hello Pearl,
Can you remove the changes to OpenSees.sln from the PR?
Thanks!
Michael

**Comment by p-ranchal** [comment]
Dear @mhscott,
All done 👍
Thanks,
Pearl

**Comment by mhscott** [comment]
Why do you divide Ep/Em in sendSelf, then multiply in recvSelf?  Why not just send Ep?

**Comment by p-ranchal** [comment]
I guess I was just trying to be consistent with the constructor signature. I will modify to send `Ep` next time I add a stiffness modifier parameter.

**Pull request merged**